components/form-builder/app/ClassificationSelect.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { cn } from "@lib/utils";
+import { ProtectedIcon } from "@formbuilder/icons";
+
+const classificationOptions = [
+  { value: "Unclassified", en: "Unclassified", fr: "Non classifié" },
```

Comment:
Curious why a hardcoded string (vs. i18n)?

Reply:
Mostly a TypeScript thing so we can declare the en / fr keys outside of the component and not have them start as empty strings. 

<img width="619" alt="Screenshot 2023-10-11 at 12 14 52 PM" src="https://github.com/cds-snc/platform-forms-client/assets/62242/5b29c462-412b-48f5-8fdc-fdf060034a70">